test/uk/gov/hmrc/emailaddress/PlayFormFormatterSpec.scala

```diff
@@ -21,18 +21,32 @@ import org.scalatest.{Matchers, WordSpec}
 import play.api.data.FormError
 import play.api.data.Forms.optional
 
-class PlayFormFormatterSpec extends WordSpec with Matchers with PropertyChecks with EmailAddressGenerators {
+class PlayFormFormatterSpec extends WordSpec with Matchers with PropertyChecks {
+
+  val emails: List[String] = EmailAddressGenerators.randomEmailAddresses().distinct
 
   "PlayFormFormatter.emailAddress mapping" should {
 
-    "accept valid emails" in forAll(validEmailAddresses) {
-      address =>
-        emailAddressMapping.bind(Map("email" -> address)).right.get shouldBe EmailAddress(address)
+    "emails should not be empty" in {
```

Comment:
Are you testing the test code here?

Reply:
It is kind a validation of generated string test cases which is generated by me for next tests :)